plugins/browser-plugin-media/src/schemata.ts

```diff
@@ -2,7 +2,7 @@ import { MediaEventType } from './types';
 
 const MEDIA_SCHEMA_PREFIX = 'iglu:com.snowplowanalytics.snowplow.media/';
 const MEDIA_SCHEMA_SUFFIX = '/jsonschema/1-0-0';
-export const MEDIA_PLAYER_SCHEMA = MEDIA_SCHEMA_PREFIX + 'player' + MEDIA_SCHEMA_SUFFIX;
+export const MEDIA_PLAYER_SCHEMA = 'iglu:com.snowplowanalytics.snowplow/media_player/jsonschema/2-0-0';
```

Comment:
We might want to leave a comment thought, WDYT ?

Reply:
Good thinking, have added a comment!